<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>8</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QListWidget" name="categories_listWidget"/>
     <widget class="QScrollArea" name="settings_scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="noBackground" stdset="0">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="settings_scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>188</width>
         <height>353</height>
        </rect>
       </property>
       <property name="noBackground" stdset="0">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="settings_verticalLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="settings_frame">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::ExpandingFieldsGrow</enum>
           </property>
           <property name="labelAlignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="margin">
            <number>8</number>
           </property>
           <item row="1" column="0">
            <widget class="QLabel" name="placeholder_label">
             <property name="text">
              <string>No settings loaded</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="settings_buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Reset|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>categories_listWidget</tabstop>
  <tabstop>settings_scrollArea</tabstop>
  <tabstop>settings_buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
